recipe/meta.yaml

```diff
@@ -7,22 +7,38 @@ package:
 source:
   fn: v{{ version }}.tar.gz
   url: https://github.com/gflags/gflags/archive/v{{ version }}.tar.gz
-  sha1: 8bdbade9d041339dc14b4ab426e2354a5af38478
+  sha1: bf07a7926cb2ce4dd4048bdc081f9e00e91cefd6
+
+  patches:
+    - windows_cmake_install_bindir.patch       # [win]
 
 build:
   number: 0
-  skip: true  # [win]
+  # py36 since it uses vc14 like py35
+  skip: true  # [win and py36]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
```

Comment:
Should add a feature for Windows Python 3.6 for good measure (even though we are skipping it for now).